apps/mobile/src/features/send/screens/select-account.tsx

```diff
@@ -80,18 +80,20 @@ interface AccountItemProps {
 
 function AccountItem({ account, asset, onSelectAccount }: AccountItemProps) {
   const wallet = useWalletByFingerprint(account.fingerprint);
-  const balance = useAccountBalance({
+  const { btc, stx } = useAccountBalance({
     accountIndex: account.accountIndex,
     fingerprint: account.fingerprint,
-  })[asset];
-  if (balance.state !== 'success') {
+  });
+
+  if (btc.state !== 'success' || stx.state !== 'success') {
     return null;
   }
 
-  if (balance.value.quote.availableBalance.amount.isZero()) {
+  const availableBalance =
+    asset === 'stx' ? stx.value.quote.availableUnlockedBalance : btc.value.quote.availableBalance;
```

Comment:
Changing this to use `availableUnlockedBalance` if the asset is STX. Before when we were taking it from the `balance` object using a lookup `[asset]` it was always showing the `quote.availableBalance`. 

I'm not sure if this is the best way to do it as I am making some assertions for `btc.state` / `stx.state`. This may not scale well but I am unsure if we will ever support **Send*** for other currencies so I think it's sufficient here